.github/workflows/render-rmarkdown.yaml

```
@@ -0,0 +1,68 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
```

NB: I don't imagine we will start having a `master` branch in the future
```suggestion
    branches: [main]
```